include/sys/util.h

```diff
@@ -204,6 +204,13 @@ extern "C" {
 #define CLAMP(val, low, high) (((val) <= (low)) ? (low) : MIN(val, high))
 #endif
 
+/**
+ * @def IN_RANGE
+ * @brief true if @p is in the @p min - @p max range (inclusive).
```

Comment:
shouldn't it be `true if @p val is`?

Reply:
yes, you are right the "val" is missing here